wsdl/ver10/schema/onvif.xsd

```diff
@@ -3828,6 +3828,22 @@ decoding .A decoder shall decode every data it receives (according to its capabi
 		</xs:restriction>
 	</xs:simpleType>
 	<!--===============================-->
+	<xs:complexType name="EditableUserLevel">
+		<xs:sequence>
+			<xs:element name="Name" type="xs:string">
+				<xs:annotation>
+					<xs:documentation>Name of the editable user level.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Name" type="tt:StringList">
+				<xs:annotation>
```

Comment:
both parameter names are named as "Name" in EditableUserLevel.  We cannot declare two variables with same name in C/C++ structure even though their data types are different.

Reply:
Thank you for spotting this copy&paste mistake. Fixed.